CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -156,14 +156,14 @@ public FlaCudaWriter(string path, Stream IO, AudioPCMConfig pcm)
 			sample_rate = pcm.SampleRate;
 			bits_per_sample = (uint) pcm.BitsPerSample;
 
-			// flake_validate_params
+			// Flake_validate_params
 
 			_path = path;
 			_IO = IO;
 
 			windowBuffer = new float[FlaCudaWriter.MAX_BLOCKSIZE * lpc.MAX_LPC_WINDOWS];
 
-			eparams.flake_set_defaults(_compressionLevel, !_settings.GPUOnly);
+			eparams.Flake_set_defaults(_compressionLevel, !_settings.GPUOnly);

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.